docs/messaging/permissions.mdx

```diff
@@ -19,28 +19,18 @@ permissions from the device Settings UI (or via a custom UI).
 
 ## Requesting permissions
 
-As explained in the [Usage documentation](usage.mdx), permission must be requested from your users in order to display
-remote notifications from FCM, via the [`requestPermission`](!firebase_messaging.FirebaseMessaging.requestPermission) API:
+As explained in the [Usage documentation](usage.mdx), permission must be requested from your iOS users in order to display
+remote notifications from FCM, via the [`requestNotificationPermissions`](!firebase_messaging.FirebaseMessaging.requestNotificationPermissions) API:
 
 ```dart
 FirebaseMessaging messaging = FirebaseMessaging.instance;
 
-NotificationSettings settings = await messaging.requestPermission(
-  alert: true,
-  announcement: false,
-  badge: true,
-  carPlay: false,
-  criticalAlert: false,
-  provisional: false,
-  sound: true,
-);
+bool isPermissionGiven = await messaging.requestNotificationPermissions();
```

Comment:
Are you sure about this change? The current API does return a `NoficiationSettings`